### ISSUE TITLE
Windows is undefined

### ISSUE THREAD

**Issue by mherodev** [MicrosoftEdge/WebAppsDocs#40]
I followed the tutorial precisely, however I am still not seeing the Windows object exposed to the hosted web site, e.g.

![image](https://cloud.githubusercontent.com/assets/1099139/15194727/4d20e6ce-1779-11e6-8b53-34671e0679fb.png)

This is on Windows 10 with VS 2015 Community Edition. Under the Content URIs tab, I made sure to  match the tutorial and include all WinRT access.

**Comment by andypavia** [comment]
I'm just double checking. Under the Content URIs tab, did you add both http://codepen.io/ and http://*.codepen.io/?

**Comment by mherodev** [comment]
Yes, and then I did many different combinations of content URIs to try and get it to cooperate. I was fortunately able to see proof-of-function using the web app here: http://contosotravel.azurewebsites.net/

But it'd be really nice to get that codepen example working again.
